src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14180,6 +14356,41 @@ Label900: ;
 		return StringOut;
 	}
 
+	bool 
+	isNumber(std::string s)
+	{
+		bool is = true;
+		try {
+			std::stof(s);
+		} catch ( std::exception & e ) {
```

Comment:
I'm not opinionated on the strtod thought.  I trust you know the performance impact better than myself.  I was expecting this wouldn't be a big performance issue since it was happening only one time during the simulation and we do much worse grotesque performancy things during "one-time" parts of the simulation.

Out of curiosity, since you'd have to check the return value of isNumber every time anyway, how much worse is performance when using a try/catch over an if/then?